### PR TITLE
Foldseek module

### PR DESCRIPTION
**Pull request by vagkaratzas**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/foldseek/createdb/main.nf [hunk 1]
```diff
@@ -0,0 +1,65 @@
+process FOLDSEEK_CREATEDB {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::foldseek=8.ef4e960"
```
Comment (sateeshperi) [inline modules/nf-core/foldseek/createdb/main.nf:5]: recommend moduleDir and environement.yml

File: modules/nf-core/foldseek/createdb/main.nf [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    tuple val(meta), path("${meta.id}"), emit: db
+    path "versions.yml"                , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
```
Comment (sateeshperi) [inline modules/nf-core/foldseek/createdb/main.nf:23]: rm empty line here if possible

File: modules/nf-core/foldseek/easysearch/main.nf [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    foldseek \\
+        easy-search \\
+        ${pdb} \\
+        ${db}/${meta_db.id} \\
+        ${prefix}.m8 \\
+        tmp \\
```
Comment (sateeshperi) [inline modules/nf-core/foldseek/easysearch/main.nf:31]: is `tmp` a parameter ?

File: modules/nf-core/foldseek/easysearch/meta.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "foldseek_easysearch"
+description: write your description here
```
Comment (sateeshperi) [inline modules/nf-core/foldseek/easysearch/meta.yml:4]: missing description here

File: modules/nf-core/foldseek/createdb/main.nf (hunk 2 above)
Comment (sateeshperi) [reply modules/nf-core/foldseek/createdb/main.nf:23]: and also other modules

File: modules/nf-core/foldseek/easysearch/main.nf (hunk 3 above)
Comment (vagkaratzas) [reply modules/nf-core/foldseek/easysearch/main.nf]: This is a name for a tmp folder that foldseek uses for intermediate calculations. Renamed it to tmpFolder as they do in their docs: https://github.com/steineggerlab/foldseek

File: modules/nf-core/foldseek/createdb/main.nf (hunk 1 above)
Comment (vagkaratzas) [reply modules/nf-core/foldseek/createdb/main.nf]: Totally forgot. Updated

File: modules/nf-core/foldseek/easysearch/meta.yml (hunk 4 above)
Comment (vagkaratzas) [reply modules/nf-core/foldseek/easysearch/meta.yml]: Description added

**Review by sateeshperi — APPROVED** [review]

**Comment by sateeshperi** [comment]
@mashehu any idea whats going on here ?

**Pull request merged**